synapse/http/request_metrics.py

```diff
@@ -43,6 +43,25 @@
     "synapse_http_server_response_time_seconds",
     "sec",
     ["method", "servlet", "tag", "code"],
+    buckets=(
+        0.005,
+        0.01,
+        0.025,
+        0.05,
+        0.1,
+        0.25,
+        0.5,
+        1.0,
+        2.5,
+        5.0,
+        7.5,
```

Comment:
Can we drop the 7.5s bucket as well? Then we'll have only added 1 extra bucket overall.